wdio config: No matching version found for @wdio/spec-reporter@^7.0.8.

**Environment:**
 - **WebdriverIO version:** 7.1.0
 - **Mode:** N/A
 - **If WDIO Testrunner, running sync/async:** N/A
 - **Node.js version:** 14.15.4
 - **NPM version:** 6.14.10
 - **Browser name and version:** N/A
 - **Platform name and version:** Mac OS 10.15.7
 - **Additional wdio packages used (if applicable):** @wdio/wdio-config, @wdio/spec-reporter

**Config of WebdriverIO**
N/A

**Describe the bug**
When installing packages while running `npx wdio config`, the build fails when it tries to install @wdio/spec-reporter@^7.0.8 which doesn't exist

**To Reproduce**
Steps to reproduce the behavior:
1. run `npx wdio config`
2. Answer the questions (You can also pass the --yes flag)
3. Build fails

**Expected behavior**
Packages install.

**Log**
```
Installing wdio packages:
- @wdio/local-runner@^7.0.8
- @wdio/mocha-framework@^7.0.8
- @wdio/spec-reporter@^7.0.8
- wdio-chromedriver-service
- @wdio/sync@^7.0.8
- chromedriver
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @wdio/spec-reporter@^7.0.8.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

The latest available version of spec-reporter is 7.0.7.  Maybe just a timing issue?  Is 7.0.8 about to be released?

--- a/packages/wdio-cli/src/commands/config.ts
+++ b/packages/wdio-cli/src/commands/config.ts
@@ -105,9 +105,12 @@ const runConfig = async function (useYarn: boolean, yes: boolean, exit = false) 
      * ensure wdio packages have the same dist tag as cli
      */
     if (pkg._requested && pkg._requested.fetchSpec) {
-        packagesToInstall = packagesToInstall.map((p) => p.startsWith('@wdio') || ['devtools', 'webdriver', 'webdriverio'].includes(p)
-            ? `${p}@${pkg._requested.fetchSpec}`
-            : p
+        const { fetchSpec } = pkg._requested
+        packagesToInstall = packagesToInstall.map((p) =>
+            (p.startsWith('@wdio') || ['devtools', 'webdriver', 'webdriverio'].includes(p)) &&
+            (fetchSpec.match(/(v)?\d+\.\d+\.\d+/) === null)
+                ? `${p}@${fetchSpec}`
+                : p
         )
     }
 

--- a/packages/wdio-cli/tests/commands/config.test.ts
+++ b/packages/wdio-cli/tests/commands/config.test.ts
@@ -98,28 +98,49 @@ test('it should install with yarn when flag is passed', async () => {
 })
 
 describe('install compliant NPM tag packages', () => {
-    beforeAll(() => {
-        // @ts-expect-error
-        pkg.setFetchSpec('beta')
-    })
+    // @ts-expect-error
+    const setFetchSpec = (fetchSpec) => pkg.setFetchSpec(fetchSpec)
+    const args = {
+        executionMode: 'sync',
+        framework: '@wdio/mocha-framework$--$mocha',
+        reporters: [],
+        services: [
+            '@wdio/crossbrowsertesting-service$--$crossbrowsertesting',
+            'wdio-lambdatest-service$--$lambdatest'
+        ],
+        generateTestFiles: false,
+        isUsingCompiler: 'TypeScript (https://www.typescriptlang.org/)'
+    }
 
-    test('it should install tagged version if cli is tagged', async () => {
-        (inquirer.prompt as any as jest.Mock).mockReturnValue(Promise.resolve({
-            executionMode: 'sync',
-            framework: '@wdio/mocha-framework$--$mocha',
-            reporters: [],
-            services: [
-                '@wdio/crossbrowsertesting-service$--$crossbrowsertesting',
-                'wdio-lambdatest-service$--$lambdatest'
-            ],
-            generateTestFiles: false,
-            isUsingCompiler: 'TypeScript (https://www.typescriptlang.org/)'
-        }))
+    test('it should install tagged version if cli is tagged with beta', async () => {
+        setFetchSpec('beta');
+        (inquirer.prompt as any as jest.Mock).mockReturnValue(Promise.resolve(args))
         await handler({} as any)
         expect(consoleLogSpy.mock.calls).toMatchSnapshot()
     })
 
-    afterAll(() => {
+    test('it should install tagged version if cli is tagged with next', async () => {
+        setFetchSpec('next');
+        (inquirer.prompt as any as jest.Mock).mockReturnValue(Promise.resolve(args))
+        await handler({} as any)
+        expect(consoleLogSpy.mock.calls).toMatchSnapshot()
+    })
+
+    test('it should install tagged version if cli is tagged with latest', async () => {
+        setFetchSpec('latest');
+        (inquirer.prompt as any as jest.Mock).mockReturnValue(Promise.resolve(args))
+        await handler({} as any)
+        expect(consoleLogSpy.mock.calls).toMatchSnapshot()
+    })
+
+    test('it should not install tagged version if cli is tagged with a specific version', async () => {
+        setFetchSpec('7.0.8');
+        (inquirer.prompt as any as jest.Mock).mockReturnValue(Promise.resolve(args))
+        await handler({} as any)
+        expect(consoleLogSpy.mock.calls).toMatchSnapshot()
+    })
+
+    afterEach(() => {
         // @ts-expect-error
         pkg.clearFetchSpec()
     })

